oshmem/mca/spml/ucx/spml_ucx_component.c

```diff
@@ -132,6 +146,22 @@ static int mca_spml_ucx_component_register(void)
                                      "Use synchronized quiet on shmem_quiet or shmem_barrier_all operations",
                                      &mca_spml_ucx.synchronized_quiet);
 
+    mca_spml_ucx_param_register_ulong("nb_progress_thresh_global", 0,
+                                    "Number of nb_put or nb_get operations before ucx progress is triggered. Disabled by default (0)",
+                                    &mca_spml_ucx.nb_progress_thresh_global);
+
```

Comment:
Shouldn't the default value be non-zero? Otherwise, progress is called on every iput/iget. Right?

Reply:
No this is the global value. nb_put_thresh and nb_get_thresh inherit this value during init.
If user sets global, then both will be set otherwise if users sets individual value then only those will be used. Individual values take precedence overriding global value.
During nb_put or nb_get operations, progress is triggered only if individual values are non-zero AND progress counters is larger than the value.